src/server/engine_shard_set.cc

```diff
@@ -32,8 +32,8 @@ ABSL_FLAG(bool, cache_mode, false,
 
 // memory defragmented related flags
 ABSL_FLAG(float, mem_defrag_threshold,
-          1,  // The default now is to disable the task from running! change this to 0.05!!
-          "Minimum percentage of used memory relative to total available memory before running "
+          0.7,  // The default now is to disable the task from running! change this to 0.05!!
```

Comment:
I think that this threshold can be bit high. it would be hard reaching it. in any case if this was changed from 1 than maybe it would be good to remove this comment I did, since it can be misleading now.

Reply:
will update the comment. If we do not reach 70% of our maxmemory capacity then  we do not need to run defrag at all.
The reason for the feature is to avoid overallocating the memory beyond maxmemory capacity.